diff -urN telegram-cli/Makefile telegram-cli-mod/Makefile
--- telegram-cli/Makefile	2015-06-16 00:15:06.735779700 +0900
+++ telegram-cli-mod/Makefile	2015-06-16 00:17:10.884076800 +0900
@@ -4,9 +4,9 @@
 LDFLAGS= -L/usr/local/lib -L/usr/lib -L/usr/lib -L/usr/lib
 CPPFLAGS= -I/usr/local/include -I/usr/include -I/usr/include  -I/usr/include/python3.4m -I/usr/include
 DEFS=-DHAVE_CONFIG_H
-COMPILE_FLAGS=${CFLAGS} ${CPFLAGS} ${CPPFLAGS} ${DEFS} -Wall -Werror -Wextra -Wno-missing-field-initializers -Wno-deprecated-declarations -fno-strict-aliasing -fno-omit-frame-pointer -ggdb -Wno-unused-parameter -fPIC
+COMPILE_FLAGS=${CFLAGS} ${CPFLAGS} ${CPPFLAGS} ${DEFS} -Wall -Wextra -Wno-missing-field-initializers -Wno-deprecated-declarations -fno-strict-aliasing -fno-omit-frame-pointer -ggdb -Wno-unused-parameter -fPIC
 EXTRA_LIBS=-ljansson -lconfig -lz -levent -lm   -lreadline -llua-5.2  -lpython3.4m -lssl -lcrypto
-LOCAL_LDFLAGS=-rdynamic -ggdb -levent ${EXTRA_LIBS} -ldl -lpthread -lutil
+LOCAL_LDFLAGS=-ggdb -levent ${EXTRA_LIBS} -ldl -lpthread -lutil
 LINK_FLAGS=${LDFLAGS} ${LOCAL_LDFLAGS}
 
 DEP=dep
diff -urN telegram-cli/loop.c telegram-cli-mod/loop.c
--- telegram-cli/loop.c	2015-06-16 00:13:58.813146200 +0900
+++ telegram-cli-mod/loop.c	2015-06-16 00:17:40.441832400 +0900
@@ -383,8 +383,8 @@
   close (auth_file_fd);
 }
 
-void write_secret_chat (tgl_peer_t *_P, void *extra) {
-  struct tgl_secret_chat *P = (void *)_P;
+void write_secret_chat (tgl_peer_t *_PP, void *extra) {
+  struct tgl_secret_chat *P = (void *)_PP;
   if (tgl_get_peer_type (P->id) != TGL_PEER_ENCR_CHAT) { return; }
   if (P->state != sc_ok) { return; }
   int *a = extra;
